scdl/scdl.py

```diff
@@ -298,6 +315,9 @@ def download(user, dl_type, name):
                 download_playlist(item)
             elif dl_type == 'playlists-liked':
                 parse_url(item['playlist']['uri'])
+            elif dl_type == 'commented':
+                item=get_track_info(item['track_id'])
+                download_track(item)
```

Comment:
Of course I did...

Every commented tracks are a in the `ressources` collection. And like I said in my comment, the API for `https://api.soundcloud.com/users/{id}/comments` do not provide other information than the `track_id` in the item collection. 

So to resolve it, I overwrite the item with a new `get_track_info(track_id)` and call download_track like in your else condition.